# mocap
/cpp_planner:
  ros__parameters:
    visualize_mesh: true
    mesh_name: "mocap" # cargo_decimate, f4
    distance_from_surface: 1.5
    min_height: 0.5 # could go to 0.5
    max_height: 3.0
    min_view_redundancy: 1
    max_incidence_angle: 20.0
    max_view_distance: 2.0
    min_distance: 0.75
    camera_half_fov: 25.0
    planner_type: greedy
    tsp_method: ortools # bnb, greedy, hc (recommended), sa, ortools
    tsp_solver_allowed_time: 6.0 # used by bnb, hc, sa
    rviz_scale: 0.15
    export_to_yaml: true
